dot/parachain/backing/candidate_backing.go

```diff
@@ -71,13 +72,16 @@ type CandidateBacking struct {
 	// or explicit view for which a `Seconded` statement has been successfully imported.
 	perCandidate map[parachaintypes.CandidateHash]*perCandidateState
 	// State tracked for all active leaves, whether or not they have prospective parachains enabled.
-	perLeaf map[common.Hash]*activeLeafState
+	perLeaf map[common.Hash]*activeLeafState // TODO: Remove this field
```

Comment:
will be addressed on this PR?

Reply:
Nope, We have multiple issues to avoid using this field. eventually this field will be removed.
I will remove the todo coment from here.